templates/common/baremetal/files/baremetal-coredns-corefile.yaml

```diff
@@ -5,7 +5,7 @@ contents:
     . {
         errors
         health :18080
-        mdns {{ .Infra.Status.EtcdDiscoveryDomain }} 0 {{`{{.Cluster.Name}}`}} {{`{{.NonVirtualIP}}`}}
+        mdns {{ .ControllerConfig.DNS.Spec.BaseDomain }} 0 {{`{{.Cluster.Name}}`}} {{`{{.NonVirtualIP}}`}}
```

Comment:
There are more places in this file that uses `.Infra.Status.EtcdDiscoveryDomain.EtcdDiscoveryDomain`. They should all be replaced with `.ControllerConfig.DNS.Spec.BaseDomain`

Reply:
Agreed

Reply:
This should all be taken care of now.